src/components/OpenFileDialog/utils.js

```diff
@@ -18,65 +18,66 @@ export const AOTypeMap = {
 
 const NO_TYPE = 'NO_TYPE'
 
-const texts = {
-    [NO_TYPE]: {
-        modalTitle: i18n.t('Open'),
-        loadingText: i18n.t('Loading'),
-        errorTitle: i18n.t("Couldn't load items"),
-        errorText: i18n.t(
-            'There was a problem loading items. Try again or contact your system administrator.'
-        ),
-        noDataText: i18n.t('No items found. Create a new to get started.'),
-        noFilteredDataText: i18n.t(
-            "No items found. Try adjusting your search or filter options to find what you're looking for."
-        ),
-        newButtonLabel: i18n.t('Create new'),
-    },
+export const getTranslatedString = (type, key) => {
+    const texts = {
+        [NO_TYPE]: {
+            modalTitle: i18n.t('Open'),
+            loadingText: i18n.t('Loading'),
+            errorTitle: i18n.t("Couldn't load items"),
+            errorText: i18n.t(
+                'There was a problem loading items. Try again or contact your system administrator.'
+            ),
+            noDataText: i18n.t('No items found. Create a new to get started.'),
```

Comment:
Kind of a weird sentence (I know this was already present):  "Create a new to get started."